controllers/glance_controller.go

```diff
@@ -493,7 +493,7 @@ func (r *GlanceReconciler) reconcileInit(
 
 func (r *GlanceReconciler) reconcileNormal(ctx context.Context, instance *glancev1.Glance, helper *helper.Helper) (ctrl.Result, error) {
 	r.Log.Info(fmt.Sprintf("Reconciling Service '%s'", instance.Name))
-
+	instance.Status.ObservedGeneration = instance.Generation
```

Comment:
I would move this to be right after we reset the conditions at L175. For me this more belongs to the status initialization code than to the Normal reconcile path. While it is not likely to cause trouble I think for consistency we want to keep the observedGeneration bumped during delete too.

Reply:
yeah, no issues but +1 for consistency